test/framework/shoot_utils.go

```diff
@@ -108,6 +109,12 @@ func (f *ShootFramework) DumpState(ctx context.Context) {
 			if err := f.dumpNodes(ctx, ctxIdentifier, f.ShootClient); err != nil {
 				f.Logger.Errorf("unable to dump information of nodes from shoot %s: %s", f.Shoot.Name, err.Error())
 			}
+		} else {
+			errMsg := ""
+			if err != nil {
+				errMsg = ": " + err.Error()
+			}
+			f.Logger.Errorf("unable to dump resources from shoot %s: API server is currently not running%s", f.Shoot.Name, errMsg)
```

Comment:
```suggestion
			f.Logger.Errorf("unable to dump resources from shoot %s: API server is currently not running %s", f.Shoot.Name, errMsg)
```

Reply:
Hmm, `errMsg` is either empty or filled with `: <some error message>`...

Reply:
oops, you're right, thanks!